CHANGELOG.md

```diff
@@ -1,5 +1,92 @@
+* [mbed-fcce] Upgraded factory-configurator-client-example to v4.7.0 and pinned its dependencies.
+* [fluentbit] Added recipe to install Fluentbit 1.3.5 on the gateway for providing an open source log processor and forwarder solution.
+  * By default, Fluent Bit is configured with following input endpoints - CPU, MEM, Systemd services - edge-core, edge-proxy, pelion-relay-term, maestro, kubelet, docker and wait-for-pelion-identity. And an output endpoint HTTP to publish logs at API endpoint - `http://gateways.local:8080/v3/device-logs` (routing through edge-proxy).
+* [journald] Enabled Forward Secure Sealing (FSS) feature of systemd journal. This helps detect gateway logs file tampering.
+  * To configure Pelion Edge gateway with a sealing key and to keep track of the verification key in production setup, use Pelion Edge Provisioner (PEP) tool [v2.3.0](https://github.com/PelionIoT/pelion-edge-provisioner/releases/tag/v2.3.0).
+  * By default, the gateway is configured with persistent journal logging. To disable that, set flag `VOLATILE_LOG_DIR = "no"` in `local.conf` and also update the `Storage` in recipes-core/systemd/systemd-conf/journald.conf. Note: If you disable persistent logging, the FSS feature won't work.
+* [systemd] Configured to manage the network. By default, we disabled Maestro's network management feature.
+* [systemd] Updated the unit startup sequence and changed `Requires` directive to `Wants`. This makes the system more robust when dealing with failing services.
+
+  ```
+  network-online --> edge-core --> wait-for-pelion-identity --> edge-proxy
+                                                            --> edge-kubelet
+                                                            --> fluentbit (td-agent-bit)
+                                                            --> pelion-relay-term
+                                                            --> maestro --> devicedb
+  ```
+  
+* [deprecation] Removed deprecated services.
+   * devicejs-ng.
+   * compatible devicejs-ng protocol translators.
+* [image improvements] The "raspberrypi" supported image "console-image" has been simplified and improved.
+   * console-image previously, version 1.0 through 2.1, contained Pelion Edge + development tools including but not limited to: compliers, editors, analysis tools, stress tools, and SQA tools.  Pelion Edge version 2.2's console-image contains a minimized set of accompany software for running and testing all of Pelion Edge's software and features.  Important to note, it is not a bare minimal image that strips common Linux tools, but more so what you might expect to find on a heavy embedded device.  With this new strategy, users of the Pelion Edge image can customize the image more easily to their liking.  It is very possible to strip the image more making a more lightweight embedded OS or add more packages to make it more like the image provide previously.
```

Comment:
Is it already understood that `console-iamge` only referrs to the Yocto build?

Reply:
added a subbullet for LMP